src/components/LayoutHeader/index.tsx

```diff
@@ -28,6 +28,10 @@ const LayoutHeader: React.FC<Required<ILayoutModifiable>> = ({ modifiers }) => {
         )}
       />
       <div className={styles.header}>
+        <div className={cn(styles.alert, collapsed && styles.collapsed)}>
+          Check out the newest tool from Iterative,{' '}
```

Comment:
let's add a bit more fun here:

🚀 Check out the newest tool 

- make font bigger
- do not mention Iterative